### PR TITLE
Update version configs page

### PR DESCRIPTION
**Pull request by dave-connors-3**
## What are you changing in this pull request and why?

update docs on `version` configs for resource ymls and `config-version` for dbt_project.yml

## Checklist

Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
- [x] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)

^ this doc doesn't seem to exist?

- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/reference/project-configs/version.md [hunk 1]
```diff
@@ -16,18 +20,86 @@ version: version
 
 </File>
 
+## `.yml` property file versions
+
+A version tag in a `.yml` property file provides the control tag, which informs how dbt processes property files. 
+
+Starting in version 1.5, dbt will no longer require this configuration in your resource `.yml` files. For more on why we required this tag, see property file [FAQs](reference/configs-and-properties#faqs).
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/version.md]:
```suggestion
Starting from version 1.5, dbt will no longer require this configuration in your resource `.yml` files.  If you want to know more about why this tag was previously required, you can refer to the [property file FAQs](reference/configs-and-properties#faqs). 
```

File: website/docs/reference/project-configs/version.md [hunk 2]
```diff
@@ -16,18 +20,86 @@ version: version
...
+</Tabs>
+
 
-The version must be in a [semantic version](https://semver.org/) format, e.g. `1.0.0`
+</VersionBlock>
 
-##  `.yml` property file versions
+<VersionBlock lastVersion="1.4">
+
+dbt projects have two distinct types of the `version` tags. This field has a different meaning depending on its location.
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/version.md]:
```suggestion
dbt projects have two distinct types of `version` tags. This field has a different meaning depending on its location.
```

File: website/docs/reference/project-configs/version.md [hunk 3]
```diff
@@ -16,18 +20,86 @@ version: version
...
+
+## `dbt_project.yml` versions
+
+The version tag in a `dbt_project` file represents the version of your dbt project. Although **this is a required parameter**, it is not currently meaningfully used by dbt. The version must be in a [semantic version](https://semver.org/) format, e.g. `1.0.0`. For more on Core versions, see [About dbt Core versions](/docs/dbt-versions/core).
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/version.md]:
```suggestion
The version tag in a `dbt_project` file represents the version of your dbt project and **is a required parameter**. However, it isn't currently used in a meaningful way by dbt. The version must follow a [semantic version](https://semver.org/) format, such as 1.0.0. For more information about dbt Core versions, refer to [About dbt Core versions](/docs/dbt-versions/core).
```

File: website/docs/faqs/Project/why-version-2.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ id: why-version-2
 
 ---
 
+<VersionBlock firstVersion="1.5">
+
+Once upon a time, the structure of these `.yml` files was very different (s/o to anyone who was using dbt back then!). Adding `version: 2` allowed us to make this structure more extensible.
+
+Currently this config is not required for resource yml files. If specified, `version: 2` is the only supported version. While we don't anticipate updating yml files to `version: 3` any time soon, if we need to introduce a new structure for these files, we'll be able to do this more easily with this config.
```
Comment (mirnawong1) [inline website/docs/faqs/Project/why-version-2.md]:
```suggestion
Resource yml files do not currently require this config. We only support `version: 2` if it's specified. Although we do not expect to update yml files to `version: 3` soon, having this config will make it easier for us to introduce new structures in the future
```

**Comment by mirnawong1** [comment]
hey @dave-connors-3 ! left some suggestions so it's active voice - let me know what you think!

**Comment by dave-connors-3** [comment]
@mirnawong1 took your great suggestions!

**Review by mirnawong1 — APPROVED** [review]
amazing, lgtm @dave-connors-3 !

**Pull request merged**